### PR TITLE
[MIG] membership_prorate_variable_period: Migration to 11.0

### PR DESCRIPTION
**Pull request by chienandalu**
- [x] Depends on https://github.com/OCA/vertical-association/pull/60

Helper module for making the prorate correctly when you are dealing with
variable periods in your memberships, which functionality is provided by
the module *membership_variable_period*.

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by chienandalu** [comment]
@pedrobaeza All green
@rafaelbn Please review

File: membership_prorate_variable_period/readme/CREDITS.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[ This file is optional and contains additional credits, other than
```
Comment (pedrobaeza) [inline membership_prorate_variable_period/readme/CREDITS.rst]: Remove this file

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

File: membership_prorate_variable_period/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import _, exceptions, models
+import datetime
```
Comment (yajo) [inline membership_prorate_variable_period/models/account_invoice.py]: Fix import order

**Review by yajo — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Tested

**Pull request merged**